hardhat.config.js

```diff
@@ -2,6 +2,7 @@ require('dotenv').config();
 require('@nomicfoundation/hardhat-toolbox');
 require('solidity-coverage');
 require('solidity-docgen');
+require('hardhat-contract-sizer');
```

Comment:
Add a unit test instead? Seems easier to test changes to bring it down if it can at least compile but fail the test

Reply:
@blockchainguyy if I am not mistaken when contract size is over the limit, the solidity compiler throws warnings anyway.